### DISCUSSION TITLE
getStaticProps in Layout

### DISCUSSION THREAD

**Discussion by jurajkrivda** [vercel/next.js#17555] [Help] [answered]
Hello. Is support for getStaticProps in _app.js planned? I need to display data from my CMS in the footer (layout) of the website. What is the current workaround for this case?

**Comment by leerob** [comment, marked as answer]
**Update:** We’re excited to announce [Next.js 13](https://nextjs.org/13), which includes the new `app/` directory (beta) with support for nested layouts, colocation of data fetching, streaming, Server Components by default, and much more.

Inside the `app/` directory, there is a powerful new way to fetch data with React’s `use()` hook and the extended Web `fetch()` API. This allows you to colocate data fetching directly inside components for the most flexibility, including inside layouts.

You can leave feedback here: https://github.com/vercel/next.js/discussions/41745
